data/mods/Magiclysm/traits/attunements.json

```diff
@@ -50,6 +50,7 @@
     "valid": false,
     "description": "",
     "prereqs": [ "ANIMIST", "TECHNOMANCER" ],
+    "spells_learned": [ [ "alchemist_make_potion_bomb", 5 ], [ "alchemist_make_potion_trap", 5 ] ],
```

Comment:
Note: the name (string) needs changing to Alchemist (from Aura Mage) - ideally also the AURA_MAGE id, here and elsewhere.